Makefile

```diff
@@ -13,6 +13,7 @@ JAVA_GEN_VERSION := v0.7.1
 TESTPARALLELISM := 10
 WORKING_DIR := $(shell pwd)
 SCHEMA_TOOLS_VERSION := $(shell cat .schema-tools.version)
+PULUMI_PROVIDER_BUILD_PARALLELISM ?= -p 2
```

Comment:
Should we maintain consistency with `TESTPARALLELISM`, where we define the value only, eg `2` instead of `-p 2`?

Reply:
NVM. Just refferred to https://github.com/pulumi/ci-mgmt/pull/386 and including the flag within the variable makes it easier to unset the flag if required.

Reply:
This is something I'd like in my .bashrc hence the longer name PULUMI_PROVIDER_BUILD_PARALLELISM